app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -178,6 +184,20 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         false
     )
 
+    fun getWallpaper(): Wallpapers {
+        return Wallpapers.valueOf(currentWallpaper)
+    }
+
+    fun toggleWallpaper(): Wallpapers {
+        val wallpaper = if (currentWallpaper.isEmpty()) {
+            Wallpapers.values().first()
+        } else {
+            Wallpapers.valueOf(currentWallpaper).next()
+        }
+        currentWallpaper = wallpaper.name
+        return wallpaper
+    }
```

Comment:
Putting this function in this location feels a little awkward to me. It could potentially be combined with the `Wallpaper::next` function in the `Wallpaper` type itself, and receive ` Settings` to perform the update, so that the signature became `Wallpaper.switchToNextWallpaper(settings: Settings)`.

Reply:
This is a nice idea for further abstracting the functionality into the same class while also keeping Settings lean.

Reply:
👍🏽